pipeline: inputs: forward: style (#1725)
fluent/fluent-bit-docs: pipeline/inputs/forward.md

| `Tag_Prefix` | Prefix incoming tag with the defined value.| _none_ |
| `Tag` | Override the tag of the forwarded events with the defined value.| _none_ |
| `Shared_Key` | Shared key for secure forward authentication. | _none_ |
| `Empty_Shared_Key` | Use this option to connect to Fluentd with a zero-length shared key. | `false` |
| `Self_Hostname` | Hostname for secure forward authentication. | _none_ |
| `Security.Users` | Specify the username and password pairs for secure forward authentication. | |
| `Threaded` | Indicates whether to run this input in its own [thread](../../administration/multithreading.md#inputs). | `false` |

## Get started

To receive Forward messages, you can run the plugin from the command line or through the configuration file as shown in the following examples.

### Command line

From the command line you can let Fluent Bit listen for Forward messages with the following options:

```bash
fluent-bit -i forward -o stdout
```

> By default the service will listen an all interfaces (`0.0.0.0`) through TCP port `24224`. You can change this by passing parameters to the command:

Reviewer: ```suggestion
By default, the service listens on all interfaces (`0.0.0.0`) through TCP port `24224`. You can change this by passing parameters to the command:
```
